Buddies/Onboarding/SignUpInfoVC.swift

```diff
@@ -0,0 +1,117 @@
+//
+
+import UIKit
+import Firebase
+import Firebase
+
+class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBOutlet weak var buttonPicture: UIButton!
+    @IBAction func changePicture(_ sender: Any) {
+        let imagePicker = UIImagePickerController()
+        imagePicker.allowsEditing = true
+        imagePicker.delegate = self
+        present(imagePicker, animated: true)
+    }
+
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        
+        if let imgUrl = info[UIImagePickerController.InfoKey.imageURL] as? URL{
+            
+            let imgName = imgUrl.lastPathComponent
+            let documentDirectory = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true).first
+            let localPath = documentDirectory?.appending(imgName)
+            
+            let image = info[UIImagePickerController.InfoKey.originalImage] as! UIImage
+            let data = image.pngData()! as NSData
+            data.write(toFile: localPath!, atomically: true)
+            
+            buttonPicture.tintColor = UIColor.clear
+            
+            buttonPicture.setImage(UIImage(contentsOfFile: localPath!), for: .normal)
+
+            let photoURL = URL.init(fileURLWithPath: localPath!)
+            
+            let curUID = getAuthHandler().getUID()!
+            
+            let storagePath = "/users/" + curUID + "/profilePicture.jpg"
+            let storageRef = StorageManager.shared.storage.reference().child(storagePath)
+            
+            let uploadTask = storageRef.putFile(from: photoURL, metadata: nil) { metadata, error in
+         
+                storageRef.downloadURL { (url, error) in
+                    if let downloadURL = url{
+                    
+                    FirestoreManager.shared.db.collection("users").document(curUID).setData([
+                        "image_url": downloadURL.absoluteString
+                        ], merge: true)
+                    }else{
+                        
+                    }
+                }
+            }
+            
+            
+            uploadTask.observe(.success) { snapshot in
+                // Upload completed successfully
+                   self.dismiss(animated: true, completion: nil)
+            }
+    
+        }
+        
+    }
+    
+    @IBOutlet weak var bioText: UITextView!
+    
+    @IBAction func finishSignUp(_ sender: Any) {
+        
+        if let UID = getAuthHandler().getUID()
+        {
+            
+            
+            if let bio = bioText.text{
+                //set bio text
+                FirestoreManager.shared.db.collection("users").document(UID).setData([
```

Comment:
One thing that would be nice is if we could fill out stuff from the data model here, even if it's blank. i'd say these things:
   - uid : string
  - name : string
  - bio : string
  - email: string
_Empty array for these things:_
  - favorite_topics : string[]
  - blocked_users : string[]
  - blocked_activities : string[]
  - blocked_by : string[]
  - date_joined : date
  - location : _could just make this be fake for now_